post-start.d/60-migrate-saml-configuration-to-database.py

```diff
@@ -112,11 +114,12 @@ def main():
 
     try:
```

Comment:
It's to add some information to the administrator if the migration fails.
I can remove the try except since. I'll just remove it since it does not add much more information than reading the name of the migration script that failewd